# Author: Ehsan Moravveji (VSC - KU Leuven)
# Purpose: Brunsli searches for libbrotli*-static.a, which is no longer
#          created in Brotli v1.1.0. This patch just adapts the names of
#          expected static libraries
diff -ruN brunsli-orig/brunsli.cmake brunsli/brunsli.cmake
--- brunsli-orig/brunsli.cmake	2024-06-12 15:45:01.042655000 +0200
+++ brunsli/brunsli.cmake	2024-06-12 15:46:39.369617000 +0200
@@ -64,7 +64,7 @@
   ${BRUNSLI_DEC_HEADERS}
 )
 target_link_libraries(brunslidec-static PRIVATE
-  brotlidec-static
+  brotlidec
   brunslicommon-static
 )
 
@@ -73,7 +73,7 @@
   ${BRUNSLI_ENC_HEADERS}
 )
 target_link_libraries(brunslienc-static PRIVATE
-  brotlienc-static
+  brotlienc
   brunslicommon-static
 )
 
